File: webservices/types/event.php
<?php

class event {
  public $eventId;
  public $nameOfDevice;
  public $timeStarted;
  public $timeEnded;


  function __construct($eventId, $nameOfDevice, $timeStarted, $timeEnded) {
    $this->eventId = $eventId;
    $this->nameOfDevice = $nameOfDevice;
    $this->timeStarted = $timeStarted;
    $this->timeEnded = $timeEnded;
  }
}

File: iOS App/Tready/Tready/Model/Types/Event.swift
//
//  Event.swift
//  Tready
//
//  Created by Brian Krupp on 4/9/18.
//  Copyright © 2018 Brian Krupp. All rights reserved.
//

import Foundation

class Event: Codable {
    public let eventId:Int
    public let nameOfDevice:String
    public let timeStarted:Double
    public let timeEnded:Double
    
    init(eventId:Int, nameOfDevice:String, timeStarted:Double, timeEnded:Double) {
        self.eventId = eventId
        self.nameOfDevice = nameOfDevice
        self.timeStarted = timeStarted
        self.timeEnded = timeEnded
    }
}
